aspnetcore/tutorials/choose-web-ui.md

```diff
@@ -0,0 +1,194 @@
+
+### Considering a server rendered UI approach
+
+A web UI app that renders on the server dynamically generates the page's HTML and CSS on the server. The page arrives at the client ready to display.
+
+Benefits:
+
+* The client requirements are minimal since the server does the work of logic and page generation:
+  * Great for low-end devices and low-bandwidth connections.
+  * Allows for a broad range of browser versions at the client.
+  * Quick initial page load times.
+  * Minimal to no code to pull to the client.
+* Flexibility of access to server resources, in a protected environment. For example:
+  * Database access.
+  * Access to secrets, such as an access key used for API calls to Azure storage.
+* Static site analysis advantages, such as search engine optimization.
+
+Examples of common server rendered web UI app scenarios:
+
+* Display read-only data such as transaction lists. 
+* Display static blog pages.
+* A public-facing content management system.
+
+Drawbacks:
+
+* The cost of compute and memory use are concentrated on the server, rather than each client.
+* User interactions require a round trip to the server to generate UI updates.
+
+### Considering a client rendered UI approach
+
+A client rendered app dynamically renders web UI on the client, directly updating the browser DOM as necessary.
+
+Benefits:
+
+* Takes advantage of the capabilities of the user’s device.
+* Allows for rich interactivity that is instant, without requiring a round trip to the server. UI event handling and logic run locally on the user's device with minimal latency. Choose a client rendered application if your app's requirements include rich user interactions.
+* Supports incremental updates, saving partially completed forms or documents without the user having to select a button to submit a form.
+* Can be designed to run in a disconnected mode. Updates to the client-side model that are eventually synchronized back to the server once a connection is re-established.
+
+Examples of client rendered web UI:
+
+* An interactive dashboard.
+* An app featuring drag-and-drop functionality
+* A responsive and collaborative social app.
+
+Drawbacks:
+
+* Code for the logic has to be downloaded and executed on the client, adding to the initial load time.
+* Client requirements may exclude user's who have low-end devices, older browser versions, or low-bandwidth connections.
+
+## Choose a predominantly server rendered ASP.NET Core UI solution
+
+The following section briefly explains the ASP.NET Core web UI server rendered models available and directs next steps to get started. ASP.NET Core Razor Pages and ASP.NET Core MVC are server-based frameworks for building web apps with .NET.
+
+### ASP.NET Core Razor Pages
+
+Razor Pages is a page-based model that is easy to get started with. UI and business logic concerns are kept separate, but within the page. Razor Pages is the preferred way to create page-based or form-based apps in ASP.NET Core.
```

Comment:
```suggestion
Razor Pages is a page-based model. UI and business logic concerns are kept separate, but within the page. Razor Pages is the recommended way to create new page-based or form-based apps for developers new to ASP.NET Core. Razor Pages provides an easier starting point than ASP.NET Core MVC.
```
RP is not easy, just look through our issues. It's easier than MVC, true. HTML Helpers are not easy.
As written will generate a lot of hate issues from the MVC faithful.